src/cpu/o3/perfCCT.hh

```diff
@@ -0,0 +1,73 @@
+#ifndef __CPU_O3_PERFCCT_HH__
+#define __CPU_O3_PERFCCT_HH__
+
+#include <string>
+
+#include "base/types.hh"
+#include "cpu/o3/dyn_inst_ptr.hh"
+#include "sim/arch_db.hh"
+
+namespace gem5
+{
+namespace o3
+{
+
+
+class InstMeta
+{
+    friend class PerfCCT;
+    InstSeqNum sn;
+    std::vector<uint64_t> posTick;
+
+  public:
+    enum Pos
+    {
+        // inst pos
+        AtFetch = 0,
+        AtDecode,
+        AtRename,
+        AtDispQue,
+        AtIssueQue,
+        AtIssueArb,
+        AtIssueReadReg,
+        AtFU,
+        AtBypassVal,
+        AtWriteVal,
+        AtCommit,
+        InvalidPos
```

Comment:
Can we define this in Python's enumeration and use it when create SLQ table to avoid redundant code?